carmajava/launch/environment.launch

```diff
@@ -33,6 +33,124 @@
   <!-- World Model Updater -->
   <include file="$(find carma_wm_ctrl)/launch/carma_wm_broadcaster.launch"/>
 
-  <!-- TODO Object Tracking and World Model go here -->
+  <!-- Object Tracking and World Model -->
+
+  <!-- vision_darknet_detect -->
+  <group>
+  <include file="$(find vision_darknet_detect)/launch/vision_yolo3_detect.launch">
+  <arg name="gpu_device_id" default="0"/>
+  <arg name="score_threshold" default="0.30"/>
+  <arg name="nms_threshold" default="0.30"/>
+
+  <arg name="network_definition_file" default="$(find vision_darknet_detect)/darknet/cfg/yolov3.cfg"/>
+  <arg name="pretrained_model_file" default="$(find vision_darknet_detect)/darknet/data/yolov3.weights"/>
+  <arg name="names_file" default="$(find vision_darknet_detect)/darknet/cfg/coco.names"/>
+
+  <arg name="camera_id" default="/"/>
+  <arg name="image_src" default="$(optenv CARMA_INTR_NS)/camera/image_raw"/>
+  </include>
+  </group>
+
+<!-- vision_beyond_track -->
+  <group>
+  <include file="$(find vision_beyond_track)/launch/vision_beyond_track.launch">
+  <arg name="camera_info_src" default="$(optenv CARMA_INTR_NS)/camera/camera_info"/>
+  <arg name="objects_topic_src" default="detection/image_detector/objects"/>
+  <arg name="camera_height_calibration_params_file" default="$(find vehicle_calibration_dir)/vision_beyond_track/calibration.yaml"/>
+  </include>
+  </group>
+  
+  <!-- lidar_euclidean_cluster_detect -->
+  <group>
+  <include file="$(find lidar_euclidean_cluster_detect)/launch/lidar_euclidean_cluster_detect.launch">
+  <arg name="points_node" default="$(optenv CARMA_LOCZ_NS)/ray_ground_filter/points_no_ground" /><!--CHANGE THIS TO READ WHETHER FROM VSCAN OR POINTS_RAW -->
+  <arg name="remove_ground" default="true" />
+  <arg name="downsample_cloud" default="false" /> <!-- Apply VoxelGrid Filter with the value given by "leaf_size"-->
+  <arg name="leaf_size" default="0.1" /><!-- Voxel Grid Filter leaf size-->
+  <arg name="cluster_size_min" default="20" /><!-- Minimum number of points to consider a cluster as valid-->
+  <arg name="cluster_size_max" default="100000" /><!-- Maximum number of points to allow inside a cluster-->
+  <arg name="sync" default="false" />
+  <arg name="use_diffnormals" default="false" />
+  <arg name="pose_estimation" default="true" />
+  <arg name="clip_min_height" default="-1.3" />
+  <arg name="clip_max_height" default="0.5" />
+
+  <arg name="keep_lanes" default="false" />
+  <arg name="keep_lane_left_distance" default="5" />
+  <arg name="keep_lane_right_distance" default="5" />
+  <arg name="cluster_merge_threshold" default="1.5" />
+  <arg name="clustering_distance" default="0.75" />
+
+  <arg name="use_vector_map" default="false" />
+  <arg name="wayarea_gridmap_layer" default="wayarea" />
+
+  <arg name="output_frame" default="velodyne" />
```

Comment:
Would this be concat_velodyne on the truck? If so we need to move it to vehicle calibration.